packages/api-client/src/views/Request/ResponseSection/ResponseSection.vue

```diff
@@ -150,7 +150,9 @@ const shouldVirtualize = computed(() => {
           <!-- Virtualized Text for massive responses -->
           <ResponseBodyVirtual
             v-if="shouldVirtualize"
-            :content="props.response!.data as string" />
+            :content="props.response!.data as string"
```

Comment:
maybe we can add `v-if="typeof props.response.data === 'string'" instead of `as string`?